scenes/main.gd

```diff
@@ -7,6 +7,8 @@ var player
 var map
 var done
 var next_sector
+var hand
+var displayPopup
```

Comment:
I know we haven't written any code conventions yet, but all other variables use snake case. Let's keep it that way for consistence until we decide otherwise =).